web/portal/src/main/resources/locale/portal/webui_en.properties

```diff
@@ -254,15 +254,15 @@ UIContainerForm.tab.label.ContainerSetting=Container Setting
 UIContainerForm.tab.label.UIContainerPermission=Permissions
 UIContainerForm.tab.label.Template=Template
 UIContainerForm.tab.label.Icon=#{word.icon}
-UIContainerForm.msg.InvalidWidthHeight=You must enter a pixel or a percentage value in field "{0}".
+UIContainerForm.msg.InvalidWidthHeight=You must enter a valid css rule in field "{0}".
```

Comment:
Can you please change the labels on Crowdin instead of PR. (this change will be erased by Crowdin Sync Job next execution if not updated on Crowdin as well @srenault-meeds @Julien-Dubois-eXo  )